src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/config.jelly

```diff
@@ -31,6 +31,12 @@
     <f:entry title="${%Disable SSL verification}" field="disableSslVerification">
       <f:checkbox default="false" />
     </f:entry>
+    <f:entry title="${%Web URL Override}" field="webUrlOverride">
```

Comment:
Maybe we can call this `Web Browser URL Override` to make it clear that we're talking about browsers?